llvm/utils/lit/lit/llvm/config.py

```diff
@@ -588,7 +588,10 @@ def use_clang(
             if getattr(self.config, pp, None)
         ]
 
-        self.with_environment("LD_LIBRARY_PATH", lib_paths, append_path=True)
+        if sys.platform.startswith("aix"):
```

Comment:
Nit: I see more cases of platform.system() == "AIX" in the LIT code, could we use that instead?

Reply:
I agreed with RoboTux @RoboTux, can we try something like line 106
```
        elif platform.system() == "AIX":
            features.add("system-aix") 
```

Reply:
```suggestion
        if platform.system() == "AIX":
```